bosunj/AfriMusic
# Chore:  Setup babel

#### What does this PR do?
- Sets up Babel

#### Description of Task to be completed?
- Install and configure Babel

#### How should this be manually tested?
- `npm run build` should compile all files in `src` folder to es5 syntax
- `import` statements should not throw errors

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#170181022](https://www.pivotaltracker.com/story/show/170181022)
